cmd/podman/play_kube.go

```diff
@@ -280,6 +289,7 @@ func kubeContainerToCreateConfig(containerYAML v1.Container, runtime *libpod.Run
 	for _, e := range containerYAML.Env {
 		envs[e.Name] = e.Value
 	}
+	containerConfig.Env = envs
```

Comment:
Wait, were we just... not using the environment variables before?

Reply:
nope I just scooted it up because an older PR I did split the for loop for envs and setting Env